pkg/infrastructure/services/deployment_test.go

```diff
@@ -15,14 +15,15 @@
 package services
 
 import (
+	"testing"
+
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
 	"github.com/stretchr/testify/assert"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
-	"testing"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const defaultDataIndexImageFullTag = infrastructure.DefaultDataIndexImageNoVersion + "latest"
+const defaultDataIndexImageFullTag = "quay.io/kiegroup/" + infrastructure.DefaultDataIndexImageName + ":latest"
```

Comment:
```suggestion
var defaultDataIndexImageFullTag = infrastructure.GetDataIndexServiceDefaultImage()
```

Reply:
This is not the same. Runtime version will return 0.10.0 instead of latest and I didn't want to modify this test... What do you think?

Reply:
The tests here don't check the Data index tag itself, it is used just as a parameter for triggering the tested method.
The change above doesn't affect the tests.

Reply:
ok ok. Will change this then.